include/afl-fuzz.h

```diff
@@ -615,6 +615,7 @@ typedef struct afl_state {
 
   u8 *   ex_buf;
   size_t ex_size;
+  list_t * list_mutators;
```

Comment:
Why allocate this dynamically? Use `list_t list_mutators` instead.

Reply:
OK. Will make the changes.

Reply:
You can see usage of list.h in the test case https://github.com/AFLplusplus/AFLplusplus/blob/fix/list/test/unittests/unit_list.c
(it still fails for long lists, see issue #291 - but will be fine for this use case)